<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depositwindow</class>
 <widget class="QDialog" name="depositwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="top_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="top_dateEdit_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="show_label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>срок размещения</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="down_dateEdit_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QCheckBox" name="down_checkBox_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="top_dateEdit_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_tax_rate">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>13</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="down_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_period">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="down_1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tax_amount">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>530</y>
     <width>290</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>tax amount</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="top_checkBox_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="show_label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>налоговая ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>280</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>начисленные проценты</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Depos_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>580</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>расчитать</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>600</y>
     <width>601</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="down_checkBox_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="down_checkBox_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="top_dateEdit_1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLineEdit" name="down_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="down_dateEdit_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="show_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>сумма вклада</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_deposit_amount">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="top_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>250</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="down_dateEdit_1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="show_label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>280</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>сумма налога</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_period">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>дней</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>месяцев</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>лет</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="show_label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>периодичность выплат</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="top_checkBox_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="top_dateEdit_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QDateEdit" name="down_dateEdit_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="show_label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>280</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>сумма вклада к концу срока</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="top_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>290</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="top_checkBox_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="deposit_amount_end">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>570</y>
     <width>290</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>deposit amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="down_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="top_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="payment_periodicity">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>каждый день</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>каждую неделю</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>раз в год</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>в конце срока</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="down_checkBox_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="down_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="down_checkBox_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_percent">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="capitalization">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>капитализация процентов</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>список снятий</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="top_checkBox_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="top_dateEdit_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="show_label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>список пополнений</string>
   </property>
  </widget>
  <widget class="QLabel" name="accured_interest">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>490</y>
     <width>290</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>accrued interest</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="down_dateEdit_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2023</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QCheckBox" name="top_checkBox_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="top_1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>процентная ставка</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
